### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.9 es 4.40 (11 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#317] [Esercizi M-Z]
![es40](https://github.com/user-attachments/assets/aeb975b7-e1ec-484a-9135-ff7f325e2c4b)

**Comment by AglaiaNorza** [comment]
![es](https://github.com/user-attachments/assets/1f7841b5-e430-49e3-a464-0b0f9d0c82c5)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/06cae91c-9352-4afe-a45d-26e44dec4e6e)
